hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -6588,8 +6677,10 @@ protected RowLock getRowLockInternal(byte[] row, boolean readLock, final RowLock
       success = true;
       return result;
     } catch (InterruptedException ie) {
-      LOG.warn("Thread interrupted waiting for lock on row: {}, in region {}", rowKey,
-        getRegionInfo().getRegionNameAsString());
+      if (LOG.isDebugEnabled()) {
+        LOG.debug("Thread interrupted waiting for lock on row: {}, in region {}", rowKey,
```

Comment:
Here the interrupt may come from the checkInterrupt call, not acquire lock?

Reply:
This was an existing case where the client would get back an IIOE. I didn't change it so the current semantics are preserved. It can be changed to use throwIfInterrupted

Reply:
I mean the log message. Waiting on row lock is not the only case now?

Reply:
No the interrupt won't come from the checkInterrupt here